2025-04-18 17:22:38 [    INFO] Starting test session. Logging to: C:\Users\denni\Documents\thoughtful task manager\logs\tests\test_run_2025-04-18_17-22-38.log (conftest.py:70)
2025-04-18 17:22:38 [    INFO] Test passed: test_client_initialization (conftest.py:82)
2025-04-18 17:22:42 [    INFO] Test passed: test_connection_retry (conftest.py:82)
2025-04-18 17:22:42 [    INFO] Test passed: test_direct_api_verification (conftest.py:82)
2025-04-18 17:22:42 [    INFO] Test passed: test_model_list_command (conftest.py:82)
2025-04-18 17:22:42 [    INFO] Test passed: test_model_name_parsing (conftest.py:82)
2025-04-18 17:22:46 [    INFO] Test passed: test_model_verification_failure (conftest.py:82)
2025-04-18 17:22:49 [    INFO] Test passed: test_ollama_service_running (conftest.py:82)
2025-04-18 17:22:49 [    INFO] Test passed: test_verify_model_prioritizes_direct_api (conftest.py:82)
2025-04-18 17:22:49 [    INFO] Test session completed (conftest.py:72)
